README.md

```diff
@@ -49,6 +49,7 @@ import React, { FC, useMemo } from 'react';
 import { ConnectionProvider, WalletProvider } from '@solana/wallet-adapter-react';
 import { WalletAdapterNetwork } from '@solana/wallet-adapter-base';
 import {
+    BananaWalletAdapter,
```

Comment:
Please limit changes to adding the package only. We don't add all new wallets to the example or starter projects.